framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/builder/WorkbenchKieServerScenarioBuilderImpl.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.cloud.openshift.operator.scenario.builder;
+
+import org.kie.cloud.api.scenario.WorkbenchKieServerScenario;
+import org.kie.cloud.api.scenario.builder.WorkbenchKieServerScenarioBuilder;
+import org.kie.cloud.openshift.operator.scenario.WorkbenchKieServerScenarioImpl;
+
+public class WorkbenchKieServerScenarioBuilderImpl implements WorkbenchKieServerScenarioBuilder {
+
+    public WorkbenchKieServerScenarioBuilderImpl() {
+    }
+
+    @Override
+    public WorkbenchKieServerScenario build() {
+        return new WorkbenchKieServerScenarioImpl();
+    }
+
+    @Override
+    public WorkbenchKieServerScenarioBuilder withExternalMavenRepo(String repoUrl, String repoUserName, String repoPassword) {
+        // TODO: currently skipped to make tests running somehow, needs to be implemented
```

Comment:
(optional) If throwing UnsupportedOperationException is not acceptable, maybe at least log this as a warning so it's a bit harder to forget?